string commandId

int32 objectId